index.html

```diff
@@ -0,0 +1,104 @@
+          <a href="https://twitter.com/microverse" target="_blank" rel="noopener noreferrer" class="fa fa-twitter"></a>
+          <a href="#English" target="_blank" rel="noopener">English</a>
+          <a href="#MyPage" target="_blank" rel="noopener">My Page</a>
+          <a href="#Logout" target="_blank" rel="noopener">Logout</a>
+        </div>
+      </section>
+      <section class="white-banner">
+        <div class="logo-container">
+          <a href="/" class="logo-container-2">
+            <img src="ressources/event-image/EoW-Logo.png" alt="A red logo for the End of the World Event" class="logo">
+          </a>
+        </div>
+        <div class="menu-container">
+          <div id="overlay-screen" class="overlay">
+            <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+            <nav class="overlay-content menu">
+              <a href="./about.html">About</a>
+              <a href="#Program">Program</a>
+              <a href="#Jobs">Jobs</a>
+              <a href="#Sponsor">Sponsor</a>
+              <a href="#News">News</a>
+            </nav>
+          </div>
+          <span class="hamburger" onclick="openNav()">&#9776;</span>
+        </div>
+        <nav class="nav-menu">
+          <a href="./about.html">About</a>
+          <a href="#Program">Program</a>
+          <a href="#Jobs">Jobs</a>
+          <a href="#Sponsor">Sponsor</a>
+          <a href="#News">News</a>
+        </nav>
+        <div class="button-container">
+          <button class="header-button" type="button" name="button">CC Campaing</button>
+        </div>
+      </section>
+    </header>
+    <section class="welcome-screen">
+      <div class="white-overlay">
+        <div class="welcome-text">
+          <p class="welcome-user">Hello Beautiful People</p>
+          <h2 class="page-title">End of the World Event 2022</h2>
+          <p class="event-intro">The end is near, and the world is doing one last event to finish lives of 7 billions people this year. for the occasion amazing VIP will accompagny you along the way to leave all together. Would you rather be alone and bored, or dance like there is no tomorrow and have no regrets ?</p>
+          <h3>2022.02.22(TUE)</h3>
+          <h4>@Camp Nou, Barcelona, Spain</h4>
+        </div>
+      </div>
+    </section>
+    <section class="main-program">
+      <h3 class="title white-text">Main Program</h3>
+      <hr>
+      <div class="program">
+        <div class="program-details" id="program">
+
+        </div>
+        <p class="program-see-more">See the whole program</p>
+      </div>
+    </section>
+    <section class="featured">
+      <h3 class="title">Featured Speakers</h3>
+      <hr>
+      <div class="speakers">
+        <div class="speakers-details" id="speaker">
+
+        </div>
+
+      </div>
+    </section>
+    <section class="partners">
```

Comment:
- [ ] Please update this section and try to match more the given design:

Yours:
![Screenshot from 2021-11-16 22-50-35](https://user-images.githubusercontent.com/62705947/142071752-c70b0ff7-e9f7-48eb-aa66-cd27cbd4a358.png)
Original:

![Screenshot from 2021-11-16 22-50-31](https://user-images.githubusercontent.com/62705947/142071778-81ae085c-0846-4fb9-a67d-e857478380a2.png)

Reply:
I have changed the following : 

styles.css
line 589 
`    justify-content: center;
`
line 597
`    padding-left: 5vw;
`

Reply:
I have added the footer as welll

Reply:
Well done